tests/components/zha/test_config_flow.py

```diff
@@ -1,125 +1,222 @@
 """Tests for ZHA config flow."""
-from unittest import mock
+
+import os
+
+import serial.tools.list_ports
+import zigpy.config
 
 from homeassistant.components.zha import config_flow
-from homeassistant.components.zha.core.const import CONTROLLER, DOMAIN, ZHA_GW_RADIO
-import homeassistant.components.zha.core.registries
+from homeassistant.components.zha.core.const import CONF_RADIO_TYPE, CONTROLLER, DOMAIN
+from homeassistant.config_entries import SOURCE_USER
+from homeassistant.const import CONF_SOURCE
+from homeassistant.data_entry_flow import RESULT_TYPE_CREATE_ENTRY, RESULT_TYPE_FORM
 
-import tests.async_mock
+from tests.async_mock import AsyncMock, MagicMock, patch, sentinel
 from tests.common import MockConfigEntry
 
 
+def com_port():
+    """Mock of a serial port."""
+    port = serial.tools.list_ports_common.ListPortInfo()
+    port.serial_number = "1234"
+    port.manufacturer = "Virtual serial port"
+    port.device = "/dev/ttyUSB1234"
+    port.description = "Some serial port"
+
+    return port
+
+
+@patch("serial.tools.list_ports.comports", MagicMock(return_value=[com_port()]))
```

Comment:
`MagicMock` is used by default to patch. Remove the `new` parameter and just set `return_value` parameter.

Reply:
I've used MagicMock, because I didn't want the mock in the test parameters. Is this a strict requirrement?

Reply:
How do you mean?

Reply:
I'm talking about the default value for `new` here:
https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch

Reply:
Oh, now I understand. Ok.